test/integration/branchcuts/generated-release-gating-jobs/test1/data/release/ci-operator/config/openshift/release/openshift-release-master__ci-4.10.yaml

```diff
@@ -0,0 +1,37 @@
+releases:
```

Comment:
There is no need to separate this file into a folder named `test1`.  We need to have all the input files in one place like we have them in the release repo. And to extend different cases you need to add more configs that include those cases. The tool should run only once and generate the desired output.

Reply:
These test cases are much like the ones you can find into the `cluster-init` integration tests here: [test/integration/cluster-init](https://github.com/openshift/ci-tools/tree/master/test/integration/cluster-init). I can adhere to the same naming conventions `input` and `expected` instead of `data` and `want` and renaming `test1`, `test2`, ... `testN` to something more meaningful but, other than that, both have test cases split into several folders, each of which recreates part of the `openshift/release` repository folders structure.
If you take a look at [test/integration/cluster-init.sh](https://github.com/openshift/ci-tools/blob/master/test/integration/cluster-init.sh) you'll notice that `cluster-init` tool runs more than once.